### DISCUSSION TITLE
TuringCourses/major/numerical_analysis/

### DISCUSSION THREAD

**Discussion by giscus** [ZJU-Turing/TuringCourses#55] [General]
# TuringCourses/major/numerical_analysis/

图灵班学习指南

https://zju-turing.github.io/TuringCourses/major/numerical_analysis/

**Comment by HobbitQia** [comment]
内容还是比较多的，但是 xww 老师人很好，给分友好，期末考试开卷，课程本身也能学到很多东西。如果有志于图形学，这门课的内容还是比较重要的。

**Comment by Yaoyaolingbro** [comment]
像MIT18.330 有学长听过或者推荐相关的网课的吗？

**Reply by HobbitQia** [reply]
我在 csdiy 看到过这门课，个人认为我们计院的 NA 已经涵盖了这方面比较重要的内容，而且除了老师讲课不算很好以外已经算得上是质量不错的课程。
如果对这方面感兴趣，不满足于应用想了解理论的话，可以看看数院信计专业的课程：数值分析、数值代数、微分方程数值解（这三门课的内容加起来差不多相当于计院 NA 的内容，也有代码作业，不过会更加注重理论）

**Comment by MTDickens** [comment]
黄劲老师的 quiz 很多，本人因为习惯听智云，好几次起晚了，就错过了小测，所以智云党慎选。

另外，以下给出三条建议，希望大家能够参考执行：
1. 通过往年智云提前把小测做出来，免得到时候猝不及防（毕竟 NA 的知识点很细碎，没什么整体性，很容易一时记混/记不住）
2. 有时候黄劲老师可能会一节课测两个小测。不过一般是单个小测过于简单的情况（比如 quiz 5, 6）。
3. 有时候黄劲老师可能会因为时间来不及而选择不测小测。但是注意会顺延到之后的课程（也就是不会跳过任何小测）。

**Reply by MTDickens** [reply]
另外黄劲老师的查重非常严格。今年看群里说有 6 个人被抓了。被查重之后，补做的成绩只有原来的 80%。所以大家尽量不要抄（<del>即使要抄也让 chatgpt 改头换面一下，最好扔到 MOSS 里再做一次检测[鼻血]</del>）。
